dluhc-component-library/src/components/timetableForm/constants.ts

```diff
@@ -15,6 +16,7 @@ const INITIAL_STAGE: Stage = {
 };
 
 export const INITIAL_STATE: FormState = {
+  [LANDING_KEY]: "",
```

Comment:
We can delete this field. The landing page doesn't need it's own bit of form state, when we add uploading from a file it will set all the other fields like title description etc.